src/Xamarin.Android.Build.Tasks/Tasks/ReadResolvedSdksCache.cs

```diff
@@ -121,6 +124,7 @@ public bool RunTask ()
 				.Select (e => e.Value)
 				.ToArray ();
 			AndroidSequencePointsMode = sdk.Element ("AndroidSequencePointsMode")?.Value ?? "None";
+			LintToolPath = sdk.Element ("LintToolPath")?.Value ?? Path.Combine (AndroidSdkBuildToolsPath);
```

Comment:
Why have `Path.Combine (AndroidSdkBuildToolsPath)`? You're not combining anything.